raiden/network/proxies/token_network.py

```diff
@@ -959,20 +908,70 @@ def close(
         }
         log.debug('closeChannel called', **log_details)
 
-        checking_block = self.client.get_checking_block()
+        if signature != EMPTY_SIGNATURE:
+            canonical_identifier = CanonicalIdentifier(
+                chain_identifier=self.proxy.contract.functions.chain_id().call(),
```

Comment:
Just like the update transfer PR, this has to be fixed (the extra `chain_id` RPC call). I will leave this for another PR.